component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/EmailOtpService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.extension.identity.emailotp.common;
+
+import org.wso2.carbon.extension.identity.emailotp.common.dto.GenerationResponseDTO;
+import org.wso2.carbon.extension.identity.emailotp.common.dto.ValidationResponseDTO;
+import org.wso2.carbon.extension.identity.emailotp.common.exception.EmailOtpException;
+
+/**
+ * EmailOtpService interface.
+ */
+public interface EmailOtpService {
+
+    /**
+     * This method validates a provided OTP.
+     *
+     * @param transactionId                     UUID to track the flow.
+     * @param userId                            SCIM Id.
```

Comment:
```suggestion
     * @param userId                            SCIM User Id.
```

Reply:
fixed in 99ef19caaf9cd3d8a29b38a5d0c58726f2c97e49